src/util.js

```diff
@@ -452,10 +452,10 @@ export default {
   },
 
   getGlobal: function() {
-    if (typeof window !== 'undefined') {
-      return window;
-    } else if (typeof self !== 'undefined') {
+    if (typeof self !== 'undefined') {
       return self;
+    } else if (typeof window !== 'undefined') {
+      return window;
```

Comment:
I tested this locally. Seems the native web crypto apis were never invoked with your patch since `self.window` is set in the worker. Good thing I benchmarked locally ;)